scripts/qiita

```diff
@@ -239,6 +246,17 @@ def load_parameters(fp, table, name):
                % (table, param.id))
 
 
+@db.command()
+@click.argument('sl_out_dir', required=True,
+                type=click.Path(resolve_path=True, readable=True, exists=True))
```

Comment:
there is also a `file_okay` option for `click.Path`s, which in this case would be `False`